scalyr_agent/agent_main.py

```diff
@@ -1214,6 +1220,9 @@ def stop(self):
                       help="For status command, prints detailed information about running agent.")
     parser.add_option("", "--no-fork", action="store_true", dest="no_fork", default=False,
                       help="For the run command, does not fork the program to the background.")
+    parser.add_option("", "--force-http", action="store_true", dest="force_http", default=False,
```

Comment:
No need to for commandline option.  This should just be an option in the configuration.